appOPHD/States/GameState.cpp

```diff
@@ -165,7 +165,6 @@ void GameState::onLoadGame(const std::string& saveGameName)
 			throw std::runtime_error("Save game file does not exist: " + saveGamePath);
 		}
 		auto newMapViewState = std::make_unique<MapViewState>(*mMainReportsState.get(), saveGamePath);
-		newMapViewState->initialize();
 		mNewMapViewState = std::move(newMapViewState);
```

Comment:
This moves saved game loading out of the `try` block. The actual loading is deferred until `initialize` is called, so we kind of want that done early here. The `initializeMapViewState` isn't called until the `update` method is run, which is well outside of the `try` block.

Actually the `MapViewState::initialize` call was originally in `GameState::initializeMapViewState`, but it was moved out to support early loading of saved games.